paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PrimaryButton.kt

```diff
@@ -29,7 +38,10 @@ internal class PrimaryButton @JvmOverloads constructor(
 
     // This is the text set by the client.  The internal label text is set to this value
     // in the on ready state and it is temporarily replaced during the processing and finishing states.
-    private var externalLabel: String? = null
+    private var originalLabel: String? = null
+
+    @VisibleForTesting
+    var externalLabel: String? = null
```

Comment:
Can this be internal?

Reply:
yup you're right. fixed in my next commit.